lms/djangoapps/teams/static/teams/js/spec/views/teams_spec.js

```diff
@@ -6,102 +6,113 @@ define([
-            teamMembershipCollection = new TeamMembershipCollection(
+        var createTeamMembershipData = function(startIndex, stopIndex) {
+            var teams = createTeamData(startIndex, stopIndex);
+            return _.map(_.range(startIndex, stopIndex + 1), function (i) {
+                return {
+                    user: {
+                        'username': 'andya',
+                        'url': 'https://openedx.example.com/api/user/v1/accounts/andya'
+                    },
+                    team: teams[i-1]
+                };
+            });
+        };
+
+        var createTeamMembership = function(teamMembershipData, options) {
+            return new TeamMembershipCollection(
                 {
                     count: 11,
                     num_pages: 3,
                     current_page: 1,
                     start: 0,
-                    results: initialTeamMemberships
+                    results: teamMembershipData
                 },
-                {
-                    course_id: 'my/course/id',
-                    parse: true,
-                    url: 'api/teams/team_memberships',
-                    username: 'andya',
-                }
+                _.extend(_.extend({}, {
+                        course_id: 'my/course/id',
+                        parse: true,
+                        url: 'api/teams/team_memberships',
+                        username: 'andya',
+                        privileged: false
+                    }),
+                    options)
             );
-        });
+        };
 
-        it('can render itself with teams collection', function () {
-            teamsView = new TeamsView({
+        var verifyCards = function(view, teams) {
+            var teamCards = view.$('.team-card');
+            _.each(teams, function (team, index) {
+                var currentCard = teamCards.eq(index);
+                expect(currentCard.text()).toMatch(team.name);
+                expect(currentCard.text()).toMatch(_.object(languages)[team.language]);
+                expect(currentCard.text()).toMatch(_.object(countries)[team.country]);
+            });
+        };
+
+        var createTeamsView = function(options) {
+            return new TeamsView({
                 el: '.teams-container',
-                collection: teamCollection,
+                collection: options.teams || createTeams(createTeamData(1, 5)),
+                teamMembership: options.teamMembership || createTeamMembership(createTeamMembershipData(1, 5)),
                 teamParams: {
+                    topicID: 'test-topic',
                     countries: countries,
                     languages: languages
                 }
             }).render();
+        };
+
+        beforeEach(function () {
+            setFixtures('<div class="teams-container"></div>');
+        });
+
+        it('can render itself with team collection', function () {
+            var testTeamData = createTeamData(1, 5),
```

Comment:
Right, I'm passing in the test data so that I can verify it later. There are a number of tests that rely upon the default data just so that they can be more concise.